pkg/apiclient/argo-kube-client.go

```diff
@@ -43,7 +45,7 @@ func newArgoKubeClient(clientConfig clientcmd.ClientConfig) (context.Context, Cl
 }
 
 func (a *argoKubeClient) NewWorkflowServiceClient() workflowpkg.WorkflowServiceClient {
-	return &argoKubeWorkflowServiceClient{workflowserver.NewWorkflowServer(sqldb.ExplosiveOffloadNodeStatusRepo)}
+	return &argoKubeWorkflowServiceClient{workflowserver.NewWorkflowServer(argoKubeOffloadNodeStatusRepo)}
```

Comment:
I didn't see the benefit of this change. are you using the same variable anywhere?

Reply:
in another PR I think - I did wonder if we should actually just log the error rather than explode- thoughts